docs/source/guides/contributor/chapters/runners.rst

```diff
@@ -568,3 +568,96 @@ for runnables with kind ``foo``.
 .. literalinclude:: ../../../../../examples/nrunner/runners/avocado-runner-foo
    :language: python
    :linenos:
+
+
+Runners messages
+----------------
+
+Every runner have to send information about the run to the avocado
```

Comment:
I believe this is better: When run as part of a job, every runner has to send information about its execution status to the Avocado job.